src/util.ts

```diff
@@ -56,9 +59,9 @@ export async function waitForNode(
   waitTimeout: number,
 ): Promise<void> {
   const { Runtime } = client
```

Comment:
Curious why you stringified these functions... I'm pretty sure they get stringified downstream. Is that not the case?